src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveterminbarn/TerminbarnRepository.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.familie.ef.sak.iverksett.oppgaveterminbarn
+
+import no.nav.familie.ef.sak.repository.InsertUpdateRepository
+import no.nav.familie.ef.sak.repository.RepositoryInterface
+import org.springframework.data.jdbc.repository.query.Query
+import org.springframework.stereotype.Repository
+import java.time.LocalDate
+import java.util.UUID
+
+@Repository interface TerminbarnRepository : RepositoryInterface<TerminbarnOppgave, UUID>,
+                                             InsertUpdateRepository<TerminbarnOppgave> {
+
+    @Query
+    public fun existsByFagsakIdAndTermindato(fagsakId: UUID, termindato: LocalDate): Boolean
+
+    // language=PostgreSQL
+    @Query("""
+        SELECT b.id behandling_id, b.fagsak_id, fe.id, bb.person_ident fodselsnummer_barn, 
+          bb.fodsel_termindato termindato_barn
+        FROM gjeldende_iverksatte_behandlinger b
```

Comment:
Bør vi forvente at det finnes en andel med utbetaling etter termindatoen?


1. Barn blir dødfødt
2. Opphører stønad
3. Lager oppfølgingsoppgave fordi barn ikke blir født...?

Eller er ikke det et stort problem?

Reply:
Da blir det evt ekstra oppgaver på noen få ? Men hva skjer om vi sjekker for andeler frem i tid, og hvis en o.s opphøres før termindatoen overskytes ? Når den senere overskytes, da blir det ikke opprettet oppgave, og hvis det er et falskt terminbarn, så blir den / de første periodene før opphør evt uoppdaget ? Eller vil ikke behandlingen være en "gyldig behandling" hvis den opphøres uansett ? 

Kan vi sjekke om det finnes oppgave som har med dødfødsel å gjøre før vi oppretter ?